spec/api-app-spec.js

```diff
@@ -494,33 +493,36 @@ describe('app module', () => {
 
       app.setLoginItemSettings({openAtLogin: true, path: updateExe, args: processStartArgs})
 
-      assert.equal(app.getLoginItemSettings().openAtLogin, false)
-      assert.equal(app.getLoginItemSettings({path: updateExe, args: processStartArgs}).openAtLogin, true)
+      expect(app.getLoginItemSettings().openAtLogin).to.be.false
+      expect(app.getLoginItemSettings({path: updateExe, args: processStartArgs}).openAtLogin).to.be.true
```

Comment:
You will probably get a linter error here because of `to.be.true` =/

Reply:
yeah i'm fixing those right now!